changelog/january2025/2025-01-09-cockpit-added-custom-retention-for-cockpits-datasources.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Custom retention for Cockpit's datasources (beta)
+status: added
+author:
+    fullname: ''
+    url: 'https://slack.scaleway.com'
+date: 2025-01-09
+category: observability
+product: cockpit
+---
+
+Since January 1st 2025, you can edit the retention duration of all your logs, metrics and traces datasources, and apply a different retention period to each of them depending on your needs. Extended retention was previously managed by Cockpit's premium plans which are now deprecated. Increasing your data retention above the default values becomes free of charge during the beta phase *only*, and will be charged again in a couple of months.  
+
```

Comment:
```suggestion
Since January 1st 2025, you can [edit the retention duration](/observability/cockpit/how-to/change-data-retention/) of all your logs, metrics and traces data sources, and apply a different retention period to each of them depending on your needs. Extended retention was previously managed by Cockpit's premium plans which are now deprecated. Increasing your data retention above the default values is free of charge **during the beta phase only**, and will be charged again in a couple of months.  
```